internal/generator/backups.go

```diff
@@ -8,8 +8,8 @@ import (
 )
 
 const (
-	defaultCheckSchedule          = "M H(3-6) * * 0"
-	defaultPruneSchedule          = "M H(3-6) * * 0"
+	defaultCheckSchedule          = "M H(5-8) * * 1"
+	defaultPruneSchedule          = "M H(1-5) * * 0"
```

Comment:
Yeah fair, I can adjust it. I don't think it will be a problem.

We also have no control over this if people have defined custom schedules for backups.